Overview.src.html

```diff
@@ -2340,18 +2340,17 @@ <h2 class="no-num no-toc">Table of Contents</h2>
    0x80 to 0xFC, inclusive, set <var>pointer</var> to
    (<var>lead</var> &minus; <var>lead offset</var>) &times; 188 + <var>byte</var> &minus; <var>offset</var>.
 
-   <li><p>Let <var>code point</var> be null, if
-   <var>pointer</var> is null, and the <span>index code point</span>
-   for <var>pointer</var> in <span>index jis0208</span> otherwise.
-
    <li>
-    <p>If <var>code point</var> is null and <var>pointer</var> is in the range
-    8836 to 10528, inclusive, return a code point whose value is
-    0xE000 + <var>pointer</var> &minus; 8836.
+    <p>If <var>pointer</var> is in the range 8836 to 10528, inclusive, return a code point whose
+    value is 0xE000 + <var>pointer</var> &minus; 8836.
```

Comment:
In the spirit of the other change in the pipeline, I suggest reordering this to `0xE000 &minus; 8836 + <var>pointer</var>`.